pkgs/os-specific/linux/broadcom-sta/default.nix

```diff
@@ -35,6 +35,8 @@ stdenv.mkDerivation {
     ./linux-4.12.patch
     ./linux-4.15.patch
     ./linux-5.1.patch
+    # source: https://salsa.debian.org/Herrie82-guest/broadcom-sta/-/commit/247307926e5540ad574a17c062c8da76990d056f
+    ./linux-5.6.patch
```

Comment:
is this patch any different than the one on salsa? (one could fetch https://salsa.debian.org/Herrie82-guest/broadcom-sta/-/raw/247307926e5540ad574a17c062c8da76990d056f/debian/patches/16-linux56.patch)

Reply:
I had to change the patch to remove the amd64 prefix from the various paths.